tests/SdAiAgent/Abilities/SiteScrapeAbilityTest.php

```
@@ -0,0 +1,160 @@
+
+	public function disable_rate_limit(): int {
+		return 0;
+	}
+
+	/**
+	 * @param false|array<string,mixed>|\WP_Error $preempt Preempted value.
+	 * @param array<string,mixed>                 $args    Request args.
+	 * @param string                              $url     Request URL.
+	 * @return array<string,mixed>|false
+	 */
+	public function mock_http( $preempt, array $args, string $url ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundAfterLastUsed
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$preempt'. (undefined)

(UnusedFormalParameter)